app/src/androidTest/java/com/example/sharingang/NewItemFragmentTest.kt

```diff
@@ -49,4 +56,29 @@ class NewItemFragmentTest {
         onView(withText(firstItem)).check(matches(isDisplayed()))
         onView(withText(secondItem)).check(matches(isDisplayed()))
     }
+
+    @Test
+    fun clickingOnGetLocationDisplaysLocation() {
+        onView(withId(R.id.newItemButton)).perform(click())
+        val button = onView(withId(R.id.new_item_get_location))
+        button.check(matches(withText("Get Location")))
+        button.perform(click())
+        Thread.sleep(5000)
```

Comment:
Probably no way to avoid the sleep here, but it would be nice, if it's easy to do.

Reply:
No clue how to do it unfortunately.